pages/android/dumpsys.md

```diff
@@ -24,6 +24,6 @@
 
 `dumpsys --skip {{service}}`
 
-- Specify a timeout period in seconds (defaults to 10s):
+- Specify a [t]imeout period in seconds (`-t` default: 10s):
 
-`dumpsys -t {{seconds}}`
+`dumpsys -t {{1..infinity}}`
```

Comment:
I am not sure about this placeholder. Let's see what others think.

Reply:
I am not sure just about the range of possible values of `-t`. ;)